components/input_environmental_personal.py

```diff
@@ -325,34 +373,49 @@ def input_environmental_personal(
 @callback(
     Output(ElementsIDs.modal_custom_ensemble.value, "opened"),
     Output(ElementsIDs.clo_input.value, "value", allow_duplicate=True),
+    Output(ElementsIDs.modal_custom_ensemble_warning.value, "display"),
+    Output(ElementsIDs.modal_custom_ensemble_warning.value, "children"),
     Input(ElementsIDs.modal_custom_ensemble_value.value, "value"),
     Input(ElementsIDs.modal_custom_ensemble_open.value, "n_clicks"),
     Input(ElementsIDs.modal_custom_ensemble_close.value, "n_clicks"),
     Input(ElementsIDs.modal_custom_ensemble_submit.value, "n_clicks"),
     State(ElementsIDs.modal_custom_ensemble.value, "opened"),
+    State(ElementsIDs.MODEL_SELECTION.value, "value"),
     prevent_initial_call=True,
 )
-def handle_modal(clo_value, _nc_open, _nc_close, _nc_submit, opened):
+def handle_modal(clo_value, _nc_open, _nc_close, _nc_submit, opened, selected_model):
 
     ctx = dash.callback_context.triggered_id
 
     if ctx == ElementsIDs.modal_custom_ensemble_open.value:
-        return True, dash.no_update
+        return True, dash.no_update, "none", dash.no_update
 
     if ctx == ElementsIDs.modal_custom_ensemble_close.value:
-        return False, dash.no_update
+        return False, dash.no_update, "none", dash.no_update
 
     total_clo_value = 0
     for value in clo_value:
         total_clo_value += float(value.split("_")[0])
     total_clo_value = round(total_clo_value, 2)
 
-    if ctx in [
-        ElementsIDs.modal_custom_ensemble_submit.value,
-    ]:
-        return False, total_clo_value
+    model_info = Models[selected_model].value
+    max_clo_value = next(
+        (
+            input_info.max
+            for input_info in model_info.inputs
+            if input_info.name == "Clothing Level"
+        ),
+        None,
+    )
+
+    if total_clo_value > max_clo_value:
```

Comment:
there should be a check also on the min clo value.

Reply:
ok, I added the min clo value check
![image](https://github.com/user-attachments/assets/b5202821-d0e4-456c-8c04-aea6a1fd579a)